config/rbac/role_binding.yaml

```diff
@@ -1,14 +1,3 @@
-kind: RoleBinding
-apiVersion: rbac.authorization.k8s.io/v1
-metadata:
-  name: updateservice-operator
-subjects:
-- kind: ServiceAccount
-  name: updateservice-operator
-roleRef:
-  kind: Role
-  name: updateservice-operator
-  apiGroup: rbac.authorization.k8s.io
```

Comment:
The `roleRef` has been removed with https://github.com/openshift/cincinnati-operator/commit/3990db30086455967c2ce5e7deba4247c4fa8b14#diff-6d4c4e29ec096149781536bb9118fde8242c1901df764e3eda3019ac8503d6c7

Reply:
Maybe we want to keep the Roles and such and move away from the ClusterRoleBinding and such now that we no longer support `All namespaces on the cluster` installs (1137a5c057902, #134, moving `AllNamespaces` to `supported: false`)?

![image](https://github.com/openshift/cincinnati-operator/assets/209920/01f9e375-a919-4cdc-8955-f1dbea956498)